index.html

```diff
@@ -1,14 +1,511 @@
+                  <input type="email" id="email" name="email" placeholder="Email">
+                </div>  
+                
+                <div class="form-wrap-2">
+                  <textarea name="message" form="footer-form" placeholder="Message"></textarea>
+                </div>
+                <button class="button-pink " type="submit" form="footer-form" value="submit"><i class="far fa-envelope-open"></i>Submit, Peasant!</button>
+
+  
+            </form>  
+          
+        <!--Contact Info on the Right Side-->
+          <div>
+            <p>olore. Recusandae quam dignissimos at? Quidem aperiam a soluta! Libero dolorum temporibus ab dolor architecto incidunt cum repudiandae, repellat sunt, vel sint soluta! Voluptatum, harum!</p>
+            <section>
+              <div class="form-grid">
+
+                <!--Contact Info-->
+                <div  class="contact">
+                  <div class="icon-flex">
+                    <a href="https://www.google.com/maps/dir//Chuck+E.+Cheese,+601+Troy+Schenectady+Rd,+Latham,+NY+12110/@41.7272364,-73.5778877,8z/data=!4m9!4m8!1m0!1m5!1m1!1s0x89de0dc68bd80c73:0xd424ef525b2294bf!2m2!1d-73.7667591!2d42.7542438!3e0"><i class="fas fa-house-user fa-2x"></i></a>
+                    <p>123 Road Road<br>
+                    Bee Hive City <br>
+                    12345s</p>
+                  </div>
+
+                  <div class="icon-flex">
+                    <i class="fas fa-phone-alt fa-2x"></i>
+                    <p>(802) 254 -8832</p>
+                  </div>
+
+                  <div class="icon-flex">
+                    <a href="gmail.com"><i class="fas fa-envelope fa-2x"></i></a>
+                    <p>email@email.com</p>
+                  </div>
+              </div>
+                
+                <!--Socials-->
+                <div class="socials">
+                  <div class="icon-flex">
+                    <a href="twitter.com/Wendys"><i class="fab fa-twitter fa-2x"></i></a>
+                    <p>Twonter.com</p>
+                  </div>
+
+                  <div class="icon-flex">
+                    <a href="https://www.instagram.com/explore/tags/farts/?hl=en"><i class="fab fa-instagram fa-2x"></i></a>
+                    <p>Instaface.com</p>
+                  </div>
+
+                  <div class="icon-flex">
+                    <a href="https://dribbble.com/shots/4256554-YO-MAMA"><i class="fas fa-basketball-ball fa-2x"></i></a>
+                    <p>Drooobble.com</p>
+                  </div>
+
+                  <div class="icon-flex">
+                    <a href="www.facebook.com"><i class="fab fa-facebook-f fa-2x"></i></a>
+                    <p>Fakebook.com</p>
+                  </div>
+              </div>
+              </div>
+            </section>  
+          </div>
+          </div>
+        </div>
+      </div>
+      </section>
+    </body>

```

Comment:
Your footer looks great! I found it to be very responsive to being both stretched and squished. Nice form alert for the icebox challenge as well!